dcap-ql/Cargo.toml

```diff
@@ -25,6 +25,7 @@ required-features = ["test-sgx-flc"]
 # If set, will link to libsgx_dcap_ql. Otherwise, will use the dynamic loader
 # to access the library at runtime.
 link = ["dcap-ql-sys/link"]
+serde_support = ["serde"]
```

Comment:
Per the [manifest docs](https://doc.rust-lang.org/cargo/reference/manifest.html), this should probably be `serde-support` (dashes, not underscores).

Reply:
Actually, I think the feature should just be called `serde`. And specifying an optional dependency does this already. So, there's no need to specifically create a feature.

Reply:
I had to leave this as an explicitly stated `serde-support` feature because otherwise I can't specify feature dependencies.